_data/sessions.yml

```diff
@@ -1,59 +1,106 @@
 -
   id: 003
-  title: "Android L Animation"
-  description: "More than ever before, developing for global audience is a necessity than a luxury in today's world. Are you ready ... where to start? what does it entail? how do I do it? how to promote in local markets? This talk will address above questions for android, chrome and web developers. In our journey of the where, what and how, we will take an in-depth look at various internationalization and localization tools & strategies available to you."
-  subtype: presentation
-  speakers: [4]
+  title: "TBA"
+  description: "TBA"
+  subtype: keynote
+  speakers: [3]
   language: en
-  complexity: "Intermediate" 
-  presentation: "https://speakerdeck.com/gdglviv/dmytro-danylyk-android-l-animation"
+  complexity: "TBA"
+  presentation: "https://speakerdeck.com/...."
+  video: "//www.youtube.com/embed/..."
 -
   id: 004
-  title: "Polymer: Interacting with Google Services using nothing but HTML"
-  description: "Google has 1M APIs and services. Every API is different! Learn how you can interact with Google services without writing code! Designing your UI so it works across multiple screen sizes is no longer a nice to have, it is *the* way to build for the web. Using Polymer, we'll cover how to create robust, responsive elements that work across all platforms."
+  title: "TBA"
+  description: "TBA"
   subtype: keynote
-  speakers: [6]
-  language: uk
-  complexity: "Beginner"
-  presentation: "https://speakerdeck.com/gdglviv/mateusz-herych-guava-beyond-collections"
-  video: "//www.youtube.com/embed/v3SCiUqI1yQ"
+  speakers: [4]
+  language: en
+  complexity: "TBA"
+  presentation: "https://speakerdeck.com/...."
+  video: "//www.youtube.com/embed/..."
 -
   id: 005
-  title: "Test: Interacting with Google Services using nothing but HTML"
-  description: "Google has 1M APIs and services. Every API is different! Learn how you can interact with Google services without writing code! Designing your UI so it works across multiple screen sizes is no longer a nice to have, it is *the* way to build for the web. Using Polymer, we'll cover how to create robust, responsive elements that work across all platforms."
+  title: "TBA"
+  description: "TBA"
+  subtype: keynote
+  speakers: [5]
+  language: en
+  complexity: "TBA"
+  presentation: "https://speakerdeck.com/...."
+  video: "//www.youtube.com/embed/..."
+-
+  id: 006
+  title: "TBA"
+  description: "TBA"
+  subtype: keynote
+  speakers: [6]
+  language: en
+  complexity: "TBA"
+  presentation: "https://speakerdeck.com/...."
+  video: "//www.youtube.com/embed/..."
+-
+  id: 007
+  title: "TBA"
+  description: "TBA"
   subtype: keynote
   speakers: [7]
   language: en
-  complexity: "Expert"
-  presentation: "https://speakerdeck.com/gdglviv/michal-tajchert-wearables-views"
-  video: "//www.youtube.com/embed/v3SCiUqI1yQ"
+  complexity: "TBA"
+  presentation: "https://speakerdeck.com/...."
+  video: "//www.youtube.com/embed/..."
```

Comment:
we can add videos and slides later.
Remove for now

Reply:
done thanks :+1: